tools/cockpit.spec

```diff
@@ -331,17 +331,22 @@ The Cockpit components for managing virtual machines.
 
 %files machines -f machines.list
 
-%package ovirt
+%package machines-ovirt
 Summary: Cockpit user interface for oVirt virtual machines
 Requires: %{name}-bridge >= %{required_base}
 Requires: %{name}-system >= %{required_base}
 Requires: libvirt
 Requires: libvirt-client
+# package of old name "cockpit-ovirt" was shipped on fedora only
+if 0%{?fedora} >= 25
```

Comment:
This is missing a `%` in front of the `if`. This causes the package build failure:

    error: line 349: Unknown tag: if 026 >= 25